src/main/java/Deadline.java

```diff
@@ -0,0 +1,20 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    protected LocalDate by;
```

Comment:
This could be better encapsulated. Perhaps make it `public final`?

Reply:
I have made it private final. Thank you!